include/swift/Basic/DemangleWrappers.h

```diff
@@ -57,5 +57,5 @@ demangleTypeAsString(StringRef MangledTypeName,
 } // end namespace demangle_wrappers
 } // end namespace swift
 
-#endif // LLVM_SWIFT_BASIC_DEMANGLE_WRAPPERS_H
+#endif // SWIFT_BASIC_DEMANGLE_WRAPPERS_H
```

Comment:
I am not sure if you need a '_' in between Demangle and wrappers. I would look in swift/llvm to see what people do. IIRC they don't do that.